packages/e2e-test-utils/src/install-theme.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Internal dependencies
+ */
+import { switchUserToAdmin } from './switch-user-to-admin';
+import { switchUserToTest } from './switch-user-to-test';
+import { visitAdminPage } from './visit-admin-page';
+import { isThemeInstalled } from './theme-installed';
+
+/**
+ * Installs a theme from the WP.org repository.
+ *
+ * @param {string} slug        Theme slug.
+ * @param {string?} searchTerm Search term to use if the theme is not findable by its slug.
+ */
+export async function installTheme( slug, searchTerm ) {
```

Comment:
Similar idea as with `deleteTheme` what if we make `searchTerm` part of options config.

```suggestion
export async function installTheme( slug, { searchTerm } ) {
```

What do you think?